sh/downgrade-db-2-7-to-2-6.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -euo pipefail
+
+NAMESPACE=kube-system
+DS_NAME=storageos-downgrade-database
```

Comment:
We should probably add an option for OCP as well turning kubectl to oc. A lot of ocp users don't have kubectl.

Reply:
I've added the CLI tool as a variable